ebu_tt_live/bindings/converters/ebutt3_ebuttd.py

```diff
@@ -223,7 +329,15 @@ def convert_children(self, element, dataset):
                 output.append(copy.deepcopy(item.value))
             elif isinstance(item, ElementContent):
                 conv_elem = self.convert_element(item.value, dataset)
+
                 if conv_elem is not None:
+                    if isinstance(item.value, StyledElementMixin) and not isinstance(item.value, style_type):
+                        self._fix_fontsize(
```

Comment:
Since the behaviour is not included in the spec I called it _fix_fontsize because it is exactly what it does. It caters for the unthought complication of converting fontSizes and create new more granular styles for them to work around the percentage inheritance implication.